pom.xml

```diff
@@ -729,6 +729,8 @@
             <skip>${surefire.skipFirstPart}</skip>
             <forkCount>${surefire.firstPartForkCount}</forkCount>
             <reuseForks>false</reuseForks>
+            <reportsDirectory>${surefire.reportsDirectory}</reportsDirectory>
+            <tempDir>${surefire.tempDir}</tempDir>
```

Comment:
Hey @markrmiller  I don't think this variable is defined anywhere. Missing some other change someplace?

Reply:
It was initially just meant to be harmless - it allows you to override via system property but does not yet try to manage tmp dirs internally. I'll file a small follow up issue to look at that. Just wanted fast, safe way to override without patching.

Reply:
According to the docs, it can already be overridden via the `tempDir` property. that not working for you?
http://maven.apache.org/surefire/maven-surefire-plugin/test-mojo.html#tempDir